README.md

```diff
@@ -1,81 +1,72 @@
 # Taiga
 
-Taiga is a framework for generalized shielded state transitions. 
-This repository contains the implementation of Taiga in Rust.
+Taiga is a framework for generalized shielded state transitions. This repository contains the implementation of Taiga in Rust.
 
-⚠️ It is a WIP project and cannot be used in production yet ⚠️
+⚠️ Taiga is WIP and cannot be used in production yet ⚠️
 
-# Introduction
+![Taiga at the bottom, as a foundation for Taiga applications, and their state is stored in Notes.](./book/src/images/Intro_UTXO.png)
 
-Taiga is a shielded state transition protocol that allows applications 
-built on top of it to enjoy the advantages of fully shielded multi-party 
-state transitions 
-(hiding the application type, the data associated with it, involved 
-parties, etc) without giving up the application complexity.
+## Taiga in 500 Words
 
-![img.png](./book/src/images/intro_taiga_app.png)
+Taiga is a state transition protocol that allows applications built on top of it to enjoy the advantages of fully shielded multi-party state transitions (hiding the application type, the data associated with it, involved parties, etc.) without giving up the application complexity. Although an independent project, Taiga is intended to be at the heart of Anoma. Conceptually, Taiga can be seen as an operating system for execution of Anoma programs. In practice, Taiga is a set of APIs for creating notes, intents, transactions – for both shielded and transparent execution; creating and verifying transaction validity proofs; computing the state changes produced by transactions; de-/serializing all of the above.
 
-[**Validity predicates**](./book/src/validity-predicates.md) are a key 
-component of 
-Taiga - applications built on top of Taiga use them to express the 
-application rules, 
-and Taiga makes sure that the rules are being followed with the help of 
-the [Action circuit](./book/src/action.md).
+A [Note](./book/src/notes.md) represents an Anoma resource in Taiga. Notes are immutable and have a "denomination" and a non-negative value, among other fields. Every note has several executable programs (often referred to as [Validity Predicates](./book/src/validity-predicates.md) or VPs) associated with it. Input for programs is the notes plus a number of additional public and private inputs to the VPs. Taiga programs result in a state change, which includes a list of destroyed (or "nullified") notes and a list of newly created notes. Any existing note being input into a program gets destroyed. Destroying a note means revealing its nullifier, a secret value bound to the note. Note ownership is determined by knowledge of the nullifier plus arbitrary logic in a VP associated with the note. Creating a note means computing its commitment (a hash), and adding it to a merkle tree. The global state of Anoma is a hashset of note nullifiers and a commitment tree.
```

Comment:
I would explain in a couple of words what you mean by "denomination", especially since you use " "

Reply:
> Every note has several executable programs

Doesn't have to be several, unless several can mean just one too

Reply:
> Input for programs is the notes

What notes and what is the purpose of those programs?

Reply:
> Taiga programs result in a state change,

Kinda vague enough to count as correct but still gives an incorrect vibe. They validate the proposed state change, don't create it

Reply:
I feel like we should switch to "consume" instead of "destroy" but it is a low priority question

Reply:
> Note ownership is determined by knowledge of the nullifier plus arbitrary logic in a VP associated with the note.

Questionable. The difference is also somewhat vague, but I would say that note ownership = knowledge of the nullifier (although we might want to clarify the difference between the case when the nullifier is published and is known to everyone), but owning a note alone doesn't allow you to induce the state change (in that sense, the state change is "owned" by a VP, because in the end VPs define if the state is acceptable)

Reply:
To address some of your questions, the purpose of this particular text is to be short and provide an intuitive overview of the whole lifecycle rather than be fully technically correct (we have specs for that).

> They validate the proposed state change, don't create it
Not since transparent execution is part of Taiga too. Transparent VPs can create state changes.

> switch to "consume" instead of "destroy"
Maybe in the specs, but this was a point that I didn't understand for a while in the beginning that "spent" or "consumed" notes essentially get destroyed.

> I would say that note ownership = knowledge of the nullifier
I meant an (optional) user VP, which would provide an additional factor for authorizing spending the note.

Reply:
> Not since transparent execution is part of Taiga too. Transparent VPs can create state changes.

Hmm interesting. The whole idea of state change created by a predicate doesn't make sense to me, are transparent predicates still predicates?

> Maybe in the specs, but this was a point that I didn't understand for a while in the beginning that "spent" or "consumed" notes essentially get destroyed.

I get what you mean, but this phrasing also confuses people. They think it literally stops existing, but it just becomes invalid (yet sort of exists in the form of cm/nf being stored). We can say destroyed for now, people are probably confused no matter what word we use 😅

> I meant an (optional) user VP, which would provide an additional factor for authorizing spending the note.

I'm really nitpicking here, we don't have to specify this detail in the overview if it hinders more than it helps 🙂

Reply:
> are transparent predicates still predicates

We will know when transparent execution spec is finished! :D

> becomes invalid

I can use the word "invalidated" probably